api/descriptions/policies/vulnerability_images_get.md

```diff
@@ -1,9 +1,96 @@
-Retrieves the rules that make up your vulnerability policy for container images.
+Retrieves the vulnerability rules for when container images with vulnerabilities are detected, and triggers alerts or blocks deployments.
```

Comment:
@KICKdesigns How about something like this:

```
Retrieves the rules in your vulnerability policy for container deployments (running containers).
Corresponds to the policy table in the Console UI in <b>Defend > Vulnerabilities > Images > Deployed</b>.
```

For each of these policy endpoints, let's map it back to where it's defined in the UI, and then rely on the main docs to explain what the rules can do.
This way, we can focus on describing how to interface with the API.